common/lc_library.cpp

```diff
@@ -1670,6 +1708,464 @@ bool lcPiecesLibrary::LoadPrimitive(lcLibraryPrimitive* Primitive)
 	return true;
 }
 
+void lcPiecesLibrary::SetLegoStyleStudFiles(bool Enable, bool Reload)
+{
+	auto SetFile = [this](QStringList &Files, const QStringList &Content, const QString &FileName)
+	{
+		const QLatin1String LineEnding("\r\n");
+		QByteArray FileData;
+
+		foreach (const QString &String, Content)
+		{
+			FileData.append(String.toLatin1().constData() + LineEnding);
```

Comment:
This line is weird and the compiler warns about it, I think you don't need the toLatin1().constData() part.

Reply:
Yep, I changed the lines from Qlatin1String to QString (so I can manipulate the stud colour) and I did not change the LineEnding var. This is just temp anyway for the PoC.